### ISSUE TITLE
Rereplication

### ISSUE THREAD

**Issue by torognes** [torognes/vsearch#151]
In some cases it could be useful with a command for rereplication (or replication or duplication or un-dereplication or whatever) of a dataset that has been dereplicated. It should restore multiple identical copies of sequence according to the orginal abundance.

**Comment by frederic-mahe** [comment]
It could be useful. Meanwhile, for users that might be interested, here is a way to do it in a bash shell, for a fasta file with swarm's abundance annotations:
```sh
awk 'BEGIN {RS = "^>" ; FS = "[_\n]"} {for (i = 1 ; i <= $2 ; i++) {print ">"$1"_1\n"$3} }' dereplicated.fasta > replicated.fasta
```
or for a fasta file with usearch style abundance annotations:
```sh
awk 'BEGIN {RS = "^>" ; FS = "[;=\n]"} {for (i = 1 ; i <= $3 ; i++) {print ">"$1";size=1;\n"$5} }' dereplicated.fasta > replicated.fasta
```

**Comment by colinbrislawn** [comment]
People may find this useful. Qiime has something similar:
http://qiime.org/scripts/inflate_denoiser_output.html

On the other hand, this is a kind of an anti-feature. It could confuse novice users because it does not truly 'undo' dereplication; at some point I will have to explain why `--rereplicate` did not return all the unique Qiime sample IDs to someone's seqs.fna file. Also, support for a consistent abundance annotation may encourage other software devs to add in this feature if their software, which is better than having to rereplicate every data set we want to use.

**Comment by torognes** [comment]
This issue should be solved in version 1.10.0 just released.
